examples/ethersv5-ts-esm/src/index.ts

```diff
@@ -1,39 +1,83 @@
-import { wrap } from "@oasisprotocol/sapphire-paratime";
+import { wrapEthersProvider, wrapEthersSigner, wrap } from "@oasisprotocol/sapphire-paratime";
 
-import { BigNumber, ContractFactory, ethers } from "ethers";
+import { ContractFactory, Wallet, providers, Contract, ethers } from "ethers";
 
-import { TestErc20Token } from "./TestErc20Token.js";
 import { assert } from "console";
 
+import OmnibusJSON from "../../../contracts/artifacts/contracts/tests/Omnibus.sol/Omnibus.json" assert { type: "json" };
+
+async function testTheContract(contract:Contract, signerAddr?:string)
+{
+    const addr = await contract.callStatic["testSignedQueries()"]!();
+    if( signerAddr ) {
+        assert( addr == signerAddr );
```

Comment:
You might want https://eslint.org/docs/latest/rules/eqeqeq

Reply:
Currently reviewing these

```
/src/sapphire-paratime/clients/js/lib/esm/cipher.js
  124:48  error  This generator function does not have 'yield'  require-yield
  129:48  error  This generator function does not have 'yield'  require-yield
  134:48  error  This generator function does not have 'yield'  require-yield
  167:48  error  This generator function does not have 'yield'  require-yield
  174:48  error  This generator function does not have 'yield'  require-yield
  187:48  error  This generator function does not have 'yield'  require-yield
  192:48  error  This generator function does not have 'yield'  require-yield
  217:65  error  This generator function does not have 'yield'  require-yield

/src/sapphire-paratime/clients/js/lib/esm/compat.js
   14:11   warning  Expected '!==' and instead saw '!='  eqeqeq
   15:25   error    'p' is already defined               no-redeclare
  300:70   warning  Expected '!==' and instead saw '!='  eqeqeq
  349:16   warning  Expected '!==' and instead saw '!='  eqeqeq
  413:122  error    'globalThis' is not defined          no-undef
  413:145  error    'globalThis' is not defined          no-undef
  413:178  error    'globalThis' is not defined          no-undef
  421:18   warning  Expected '===' and instead saw '=='  eqeqeq

/src/sapphire-paratime/clients/js/scripts/proxy.ts
   45:18  warning  Expected '!==' and instead saw '!='  eqeqeq
   71:23  warning  Expected '===' and instead saw '=='  eqeqeq
   74:21  warning  Expected '===' and instead saw '=='  eqeqeq
   75:21  warning  Expected '===' and instead saw '=='  eqeqeq
   85:34  warning  Expected '===' and instead saw '=='  eqeqeq
   88:29  warning  Expected '===' and instead saw '=='  eqeqeq
  112:30  warning  Expected '===' and instead saw '=='  eqeqeq
  117:27  warning  Expected '===' and instead saw '=='  eqeqeq

/src/sapphire-paratime/clients/js/src/compat.ts
  465:65  warning  Expected '!==' and instead saw '!='  eqeqeq
  529:14  warning  Expected '!==' and instead saw '!='  eqeqeq
  624:16  warning  Expected '===' and instead saw '=='  eqeqeq

/src/sapphire-paratime/clients/js/test/compat.spec.ts
  126:23  warning  Expected '===' and instead saw '=='  eqeqeq
  189:26  warning  Expected '===' and instead saw '=='  eqeqeq
```